Samples/SampleProjectLab/MeadowApp.cs

```diff
@@ -21,7 +21,7 @@ public override Task Run()
         {
             LogService.Log.Trace("Initializing Glade game engine...");
             glade = new Game();
-            glade.Initialize(display, 2, EngineMode.GameLoop, RotationType._90Degrees);
+            glade.Initialize(display, 1, EngineMode.GameLoop, RotationType._90Degrees);
```

Comment:
See comment previous comment about display scale. Not strongly opinionated here, just want to make sure this was purposeful.

Reply:
Resolved via slack convo. We'll leave this like this for now as we work through perf issues.